index.html

```diff
@@ -0,0 +1,54 @@
+  </head>
+  <body>
+    <div class = 'container'>
+      <div class = 'header'>
+        <h1 class = 'title'>HANGMAN</h1>
+        <button class = 'btn'>START GAME</button>
+      </div>
+      <div class = 'game-wrapper'>
+        <div class = 'image-container'>
+          <img class = 'image' src="img/s0.jpg">
+        </div>
+        <div class = 'letter-container a border-bottom border-right'><p>A</p></div>
+        <div class = 'letter-container b border-bottom border-right border-left'><p>B</p></div>
+        <div class = 'letter-container c border-bottom border-right border-left'><p>C</p></div>
+        <div class = 'letter-container d border-bottom border-right border-left'><p>D</p></div>
+        <div class = 'letter-container e border-bottom border-right border-left'><p>E</p></div>
+        <div class = 'letter-container f border-bottom border-right border-left'><p>F</p></div>
+        <div class = 'letter-container g border-bottom border-left'><p>G</p></div>
+        <div class = 'letter-container h border-bottom border-left border-top'><p>H</p></div>
+        <div class = 'letter-container i border-bottom border-left border-top'><p>I</p></div>
+        <div class = 'letter-container j border-bottom border-left border-top'><p>J</p></div>
+        <div class = 'letter-container k border-bottom border-left border-top'><p>K</p></div>
+        <div class = 'letter-container l border-bottom border-left border-top'><p>L</p></div>
+        <div class = 'letter-container m border-bottom border-left border-top'><p>M</p></div>
+        <div class = 'letter-container n border-left border-top'><p>N</p></div>
+        <div class = 'letter-container o border-left border-top border-right'><p>O</p></div>
+        <div class = 'letter-container p border-left border-top border-right'><p>P</p></div>
+        <div class = 'letter-container q border-left border-top border-right'><p>Q</p></div>
+        <div class = 'letter-container r border-left border-top border-right'><p>R</p></div>
+        <div class = 'letter-container s border-left border-top border-right'><p>S</p></div>
+        <div class = 'letter-container t border-top border-right'><p>T</p></div>
+        <div class = 'letter-container u border-top border-right border-bottom'><p>U</p></div>
+        <div class = 'letter-container v border-top border-right border-bottom'><p>V</p></div>
+        <div class = 'letter-container w border-top border-right border-bottom'><p>W</p></div>
+        <div class = 'letter-container x border-top border-right border-bottom'><p>X</p></div>
+        <div class = 'letter-container y border-top border-right border-bottom'><p>Y</p></div>
+        <div class = 'letter-container z border-top border-right border-bottom'><p>Z</p></div>
+      </div>
+      <div class = 'sentence-wrapper'>
+        <h3 class = 'sentence'>This is only a temporary sentence This is only a temporary sentence</h3>
+      </div>
+    </div>
+    <script src="app.js" async defer></script>
```

Comment:
are you aware what does it mean `async` and `defer` ? I don't think you need it here ;)